tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java

```diff
@@ -88,7 +89,17 @@ public IStatus doCreateNewProject(
         final String displayName = op.getDisplayName().content( true );
         final boolean separateJRE = true;
 
-        final SDK sdk = SDKUtil.getWorkspaceSDK();
+        SDK sdk = SDKUtil.getWorkspaceSDK();
+
+        if ( sdk == null )
+        {
+            sdk = SDKUtil.createSDKFromLocation( PathBridge.create( op.getSdkLocation().content() ) );
+        }
+
+        if ( sdk == null || !sdk.validate().isOK() )
```

Comment:
if null then create new status.

if not null but sdk.validate has error return that status